### PR TITLE
V0.1.0 release candidate

### LINKED ISSUE #22: add package.json
should include, at minimum

- [ ] name
- [ ] description
- [ ] version
- [ ] license

### LINKED ISSUE #34: replace battery health w/ AODS data
#32 should be AODS data, not battery health, as that's covered in the health packet

### PR DESCRIPTION
**Pull request by capsulecorplab**
closes #34 
closes #22

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "name": "dilophos",
+    "version": "0.1.0",
+    "description": "Dilophosaurus Operating System (DilophOS)",
```
Comment (kchiu002) [inline package.json:4]: Are we not adding "keywords", "repository", and "homepage" to this package.json?

**Review by kchiu002 — COMMENTED** [review]
See line comment.

File: CHANGELOG.adoc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+=== Initial Release 0.1.0 - Critical Design Review (CDR)
```
Comment (capsulecorplab) [inline CHANGELOG.adoc]: Going to change this to "pre-release"

File: package.json (hunk 1 above)
Comment (capsulecorplab) [reply package.json:4]: Ah, good catch

**Review by kchiu002 — APPROVED** [review]
LGTM

**Pull request merged**